corehq/apps/app_manager/models.py

```diff
@@ -5870,7 +5870,9 @@ def save(self, *args, **kwargs):
     @property
     @memoized
```

Comment:
You can drop `@memoized` now that it's cached in `self._build`

Reply:
https://github.com/dimagi/commcare-hq/pull/24692/commits/4bdc44cc3dbff1fb72c6447228a559fbac4682da